### PR TITLE
fixed previous PR for team 4941

### PR DESCRIPTION
**Pull request by dvtate**
The links are no longer wrapped in <a> tags.
I think this is all I needed to change. If there's any other recommendations/requirements please tell me (I'll probably be updating this further after build season).

### CODE REVIEW

File: _frc/4900/4941.md [hunk 1]
```diff
@@ -9,6 +9,23 @@ team:
   sponsors:
     - Howard High School
   links:
-    Website: 
+    Code: https://github.com/robobibb/
+    Twitter(@FRC4941): https://twitter.com/FRC4941
```
Comment (virtuald) [inline _frc/4900/4941.md]: Yeah... so this part of the file is YAML, so it has to be strictly formatted. It should be `Twitter:`

File: _frc/4900/4941.md [hunk 2]
```diff
@@ -9,6 +9,23 @@ team:
...
+    Facebook: https://fb.com/teamrobobibb/
+robot_code:
+  2014 - 2015:
```
Comment (virtuald) [inline _frc/4900/4941.md]: Same thing here, 2014-2015 isn't a valid YAML key, just use a single parameter.

Maybe in the summer we'll make this easier to do, sorry about that!

**Review by virtuald — CHANGES_REQUESTED** [review]

File: _frc/4900/4941.md (hunk 2 above)
Comment (dvtate) [reply _frc/4900/4941.md]: Would having different names for the parts of the code be allowed as is done here?
```
  2017:
  - Main-Robot:
    - https://github.com/robobibb/main-bot-2017
    - C++
  - Practice-Robot:
    - https://github.com/robobibb/test-bot-2017
    - C++
```

**Comment by virtuald** [comment]
I went ahead and fixed the formatting for you and pushed it in a separate commit.

**Comment by dvtate** [comment]
Ok thx

**Pull request closed without merging**